docs/ember/template-only-components.md

```diff
@@ -26,6 +26,23 @@ declare module '@glint/environment-ember-loose/registry' {
 
 Note that the runtime content of this module (effectively `export default templateOnlyComponent();`) is exactly what Ember generates at build time when creating a backing module for a template-only component.
 
+Additionally, you can create a backing module using the `TOC` type, which is a convenience alias for `templateOnlyComponent`.
+
+```typescript
```

Comment:
adding an example file name here would be good, I think

Reply:
Makes sense to me! I added `app/components/shout.ts` as a filename